tests/integ/modin/groupby/test_groupby_basic_agg.py

```diff
@@ -388,6 +388,16 @@ def test_string_sum(data):
     )
 
 
+@sql_count_checker(query_count=0)
+def test_groupby_sum_string_argument_exception():
+    snow_df = pd.DataFrame([[1, 2, 3], [4, 5, 6]], columns=["key_col", "col1", "col2"])
+    with pytest.raises(
+        ValueError,
+        match="GroupBy aggregations like sum take a numeric_only argument that needs to be a bool, but a str value was passed in.",
```

Comment:
The match would have to be fixed here too sorry, would import and use `re.escape`

Reply:
No worries!